src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1202,6 +1192,42 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
         return responseRows;
     }
 
+    private void addCommentsToResponseRow(FeedbackResponseAttributes response, FeedbackQuestionAttributes question,
```

Comment:
Although this is a private method. I believe it is worthy to write javadoc especially for `responseGiverRecipientList`.